helm_deploy/values-preprod.yaml

```diff
@@ -6,11 +6,11 @@ replicaCount: 4
 image:
   repository: quay.io/hmpps/hmpps-audit-api
   tag: latest
+  pullPolicy: Always
```

Comment:
I don't think this is right.  There should be no need to pull every time - the cached version should be good enough.  (Luckily the tag is overridden or the combination of latest / always can cause pain!)